src/WorkBC.Importers.Federal/Services/XmlImportService.cs

```diff
@@ -497,25 +497,23 @@ public async Task<List<JobPosting>> GetAllJobPostingItems()
         }
 
         /// <summary>
-        ///     Get job XML if the job should not be skipped
+        ///     Get XML for english and french job details from Federal Job API
         /// </summary>
-        private async Task GetXmlContent(long jobId, bool importEnglish, bool importFrench)
+        private async Task GetXmlContent(long jobId)
         {
             _xmlDocumentEnglish = null;
             _xmlDocumentFrench = null;
-
-            if (importEnglish)
-            {
-                _xmlDocumentEnglish = await GetWebResponse($"{_federalSettings.FederalJobXmlRoot}/en/{jobId}.xml");
-            }
-
-            if (importFrench)
-            {
-                _xmlDocumentFrench = await GetWebResponse($"{_federalSettings.FederalJobXmlRoot}/fr/{jobId}.xml");
-            }
+            var tasks = new Task<XmlDocument>[2];
+            
+            tasks[0] = GetWebResponse($"{_federalSettings.FederalJobXmlRoot}/en/{jobId}.xml");
+            tasks[1] = GetWebResponse($"{_federalSettings.FederalJobXmlRoot}/fr/{jobId}.xml");
+            
+            await Task.WhenAll(tasks);
+            _xmlDocumentEnglish = tasks[0].Result;
+            _xmlDocumentFrench = tasks[1].Result;
```

Comment:
Rather than await the requests for the french and english job board descriptions sequentially, run the API calls in parallel.